app/governance/proposal/page.tsx

```diff
@@ -39,6 +55,12 @@ export default function Page() {
   const { proposals, isProposalsLoading, newVoteFlow } = useProposals({
     chainId: chainId,
   });
+
+  const { isLoading, validators, apr, userStaking, selection, transaction } =
```

Comment:
useStaking being imported ?

Reply:
useStaking is imported as we meed userStaking data we don't have a separate hook for userStaking. Will need to separate the userStaking in another hook